blob.py

```
@@ -0,0 +1,14 @@
+import os
+
+# Set the folder path here
+folder_path = "C:/Users/paul.connor/OneDrive - Keywords Studios Ltd/Pictures/New folder"
```

Pylint (3.3.7)

[convention] Constant name "folder_path" doesn't conform to UPPER_CASE naming style

(C0103)